src/xtgeo/grid3d/_grid3d_fence.py

```diff
@@ -64,12 +189,14 @@ def get_randomline(
     if zmax is None:
         zmax = self._tmp["basd"].values.max()
 
+    print("ZICREMENT", zincrement)
+
     nzsam = int((zmax - zmin) / float(zincrement)) + 1
     nsamples = xcoords.shape[0] * nzsam
 
     logger.info("Running C routine to get randomline...")
-    self._xtgformat1()
-    self._tmp["onegrid"]._xtgformat1()
+    self._set_xtgformat1()
```

Comment:
Have renamed this private function to be more explisit. Formerly
```
self._xtgformat = 2
self._xtgformat2()
```
brought some developer confusion (at least for me...)